README.md

```diff
@@ -1,9 +1,74 @@
-Seldon Core Operator
-====================
+## Seldon Core Operator - a component of the Charmed Kubeflow distribution from Canonical
 
-### Overview
 This repository hosts the Kubernetes Python Operator for Seldon Core
 (see [CharmHub](https://charmhub.io/?q=seldon-core)).
 
-The Seldon Core Operator is a Python script that wraps the latest released Seldon Core manifest,
-providing lifecycle management and handling events (install, upgrade, integrate, remove).
+The Seldon Core Operator is a Python script that wraps the latest released Seldon Core manifest, providing lifecycle management and handling events (install, upgrade, integrate, remove).
+
+### Usage
+
+The Seldon Core Operator may be deployed using the Juju command line as
+follows
+
+```sh
+$ juju deploy seldon-core
```

Comment:
Due to issue #5 we actually can't deploy seldon with its default application name, therefore the correct way of deploying it is actually:
`juju deploy seldon-core seldon-controller-manager`